tests/scattering2d/test_numpy_scattering2d.py

```diff
@@ -137,32 +137,21 @@ def test_fft(self, backend):
                       [x[0, 0] + x[0, 1] - x[1, 0] - x[1, 1],
                        x[0, 0] - x[0, 1] - x[1, 0] + x[1, 1]]])
 
-        z = backend.fft(x, direction='C2C')
```

Comment:
Why aren't we testing the `fft` function?

Reply:
has this been addressed?

Reply:
good thinking.